xbmc/GUIInfoManager.cpp

```diff
@@ -2118,6 +2118,15 @@ const infomap videoplayer[] =    {{ "title",            VIDEOPLAYER_TITLE },
 /// @{
 /// \table_start
 ///   \table_h3{ Labels, Type, Description }
+///   \table_row3{   <b>`VideoFilter`</b>,
+///                  \anchor RetroPlayer_VideoFilter
+///                  _string_,
+///     Returns the video filter of the currently-playing game.\n
+///     The following values are possible:
+///     - nearest (Nearest-neighbor, i.e. pixellate)
+///     - linear (Bilinear, i.e. smooth blur)
+///     - <path to shader> (not supported yet)
```

Comment:
Not sure we want to document options that are not yet available. Maybe mark them with a "@todo" in the code itself and hide it from the developer docs